tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -165,6 +165,49 @@ TEST(caffe2, convNHWC) {
   EXPECT_EQ(mod.getConstants().size(), 2);
 }
 
+/// Test loading ConvolutionChannelwiseQuantizedNode op from a Caffe2 model.
+/// The input is N*H*W*C (1*1*1*4), the kernel is 1,
+/// stride is 1, pad is 1, group is 2.
+TEST(caffe2, convGroupQuantized) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename(
+      GLOW_DATA_PATH
+      "tests/models/caffe2Models/conv_group_quantized_pred_net.pbtxt");
+  std::string NetWeightFilename(
+      GLOW_DATA_PATH
+      "tests/models/caffe2Models/conv_group_quantized_init_net.pbtxt");
+
+  Placeholder *output;
+  PlaceholderBindings bindings;
+
+  Tensor input(ElemKind::Int8QTy, {1, 1, 1, 4}, 1.0, 0);
+
+  // Destroy the loader after the graph is loaded since the following execution
+  // will not depend on anything from the loader.
+  {
+    Caffe2ModelLoader caffe2LD(NetDescFilename, NetWeightFilename, {"input"},
+                               {&input.getType()}, *F);
+    output = EXIT_ON_ERR(caffe2LD.getSingleOutput());
+  }
+
+  // High level check on the content of the graph. We have 1
+  // ConvolutionChannelwiseQuantizedNode and 1 save.
+  EXPECT_EQ(F->getNodes().size(), 2);
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *groupwiseConv = llvm::dyn_cast<ConvolutionChannelwiseQuantizedNode>(
```

Comment:
Perhaps check that all the params of the conv are set correctly, e.g. kernels/strides/pads/group/order/etc.?